### ISSUE TITLE
There is no check if the imported file has a default.

### ISSUE THREAD

**Issue by Guibod** [gajus/create-index#36]
Create-index is pretty dumb actually, it would try to include a file as default only if it exists and validates some naming convention. It may eventually import as default a file without default.

**Comment by gajus** [comment]
I don't think this is "dumb". Project directory structure is "dumb" if `index.js` is used with `create-index` in a directory that includes files without `export default`.

In the above case, it would be an enhancement to throw an error. In combination with the proposed [ignore configuration](https://github.com/gajus/create-index/issues/31), it would cover all use case.

**Comment by DaveWelling** [comment]
If no `export default` is found, instead could you create 
`export { * as moduleName } from './moduleName.js';`
?
